### ISSUE TITLE
Adding YASMIN-RPC Support

### ISSUE THREAD

**Issue by cyberphone** [FasterXML/jackson-future-ideas#13]
https://cyberphone.github.io/doc/web/yasmin.html#apimapping

This may either be a Spring or CXF feature.
The signature option requires that Jackson can deal with ES6 compatible serialization.

**Comment by cowtowncoder** [comment]
You need to be bit more specific explaining what kind of support you would be thinking.
Jackson does not support RPC at all (that's a layer above), although JAX-RS providers published are sort of similar I suppose.

**Comment by cyberphone** [comment]
From the Jackson horizon, ES6 compatible serialization would (minimally) be enough, RPC would be provided by Spring or CXF.  Signature support could either be provided in Jackson or in RPC frameworks.  In my own software, signatures have been integrated in the JSON tools:
https://cyberphone.github.io/doc/openkeystore/javaapi/org/webpki/json/JSONObjectWriter.html#setSignature-org.webpki.json.JSONSigner-

**Comment by cowtowncoder** [comment]
ES6 compatible serialization, outlining specifics, would fall under `jackson-core` I assume.
Would this entail something other than try to limit range of 64-bit longs and such?

**Comment by cyberphone** [comment]
The only thing missing is the conversion of Number to String according to ES6's very specific scheme.  Assuming that Jackson is able maintaining property order "as received" during serialization, all would then be there to support signing JSON objects with ease.

**Comment by cowtowncoder** [comment]
Ah. No wonder this sounded familiar:

https://github.com/FasterXML/jackson-core/issues/252

**Comment by cyberphone** [comment]
Yes, it has only grown a bit since last time with Spring or CXF integration in mind.
